### PR TITLE
include cli in coverage report

### PR DESCRIPTION
**Pull request by dholth**
### Description

Include more code in coverage report; remove unnecessary excludes.

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [x] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

File: tests/test_root.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def test_gpg_key_retrieval_with_unknown_fingerprint():
     #         testing suite we're using provides.
 
     with pytest.raises(securesystemslib.gpg.exceptions.KeyNotFoundError):
-        full_gpg_pubkey = gpg_funcs.export_pubkey(SAMPLE_UNKNOWN_FINGERPRINT)
```
Comment (dholth) [inline tests/test_root.py:101]: unused return values

File: tests/test_authentication.py [hunk 2]
```diff
@@ -177,8 +177,6 @@ def test_wrap_sign_verify_signable():
     )
 
     # Clean up a bit for the next tests.
-    new_private = loaded_new_private
-    new_public = loaded_new_public
```
Comment (dholth) [inline tests/test_authentication.py:181]: unused values

File: conda_content_trust/cli.py [hunk 3]
```diff
@@ -24,6 +25,17 @@
...
 
 def cli(args=None):
+    parser = build_parser()
+
+    args = parser.parse_args(args)
+
+    if hasattr(args, "func"):
+        return args.func(args)
+    else:
+        parser.print_help()
```
Comment (dbast) [inline conda_content_trust/cli.py:35]: can be simplified by setting `sp.required = True` in line 40/52 and then just
```suggestion
    parser.print_help()
```

Comment (dholth) [reply conda_content_trust/cli.py:35]: You're saying for all the subparsers, I need to add a required=True parameter?

Comment (dholth) [reply conda_content_trust/cli.py:35]: I added required=True and change `subcommand_name` to `subcommand`, to produce this error message when no subcommand is given `conda-content-trust: error: the following arguments are required: subcommand`

Comment (dbast) [reply conda_content_trust/cli.py]: yep adding `required=True` once as you did does the trick

**Review by dbast — APPROVED** [review]

**Pull request merged**